src/libraries/Common/tests/TestUtilities/System/LineEndingsHelper.cs

```diff
@@ -5,14 +5,9 @@ namespace System
 {
     public static class LineEndingsHelper
     {
-        public const string CompiledNewline = @"
+        private const string CompiledNewline = @"
 ";
-        public static readonly bool s_consistentNewlines = StringComparer.Ordinal.Equals(CompiledNewline, Environment.NewLine);
-
-        static public bool IsNewLineConsistent
-        {
-            get { return s_consistentNewlines; }
-        }
+        private static readonly bool s_consistentNewlines = StringComparer.Ordinal.Equals(CompiledNewline, Environment.NewLine);
```

Comment:
```suggestion
        private static readonly bool s_consistentNewlines = StringComparer.Ordinal.Equals(CompiledNewline, Environment.NewLine);
        
        static public bool IsNewLineConsistent
        {
            get { return s_consistentNewlines; }
        }
```
Looks like this removal broke other tests consuming this file.